src/table/table-role/interfaces.ts

```diff
@@ -18,3 +18,5 @@ export interface FocusedCell {
   cellElement: HTMLTableCellElement;
   element: HTMLElement;
 }
+
+export type FocusableDefinition = (() => HTMLElement) | React.RefObject<HTMLElement>;
```

Comment:
If this is our internal only API, can we agree on one format and use only the callback or only the refs?